keep-ui/components/ui/Modal.tsx

```diff
@@ -0,0 +1,49 @@
+import { Dialog, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+export default function Modal({
+  children,
+  isOpen,
+  onClose,
+}: {
+  children: React.ReactNode;
+  isOpen: boolean;
+  onClose: () => void;
+}) {
+  return (
+    <Transition appear show={isOpen} as={Fragment}>
+      <Dialog as="div" className="relative z-50" onClose={onClose}>
```

Comment:
Logic looks good. I was wondering whether we should utilize the [Tremor dialog](https://www.tremor.so/docs/ui/dialog)?

Reply:
Great idea.